container_images/pytest/README.md

```diff
@@ -63,10 +63,19 @@ See the Dockerfile for currently-installed versions.
 
 #### test-deps (optional)
 
-*Additional* dependencies to install prior to running tests. Runtime
-dependencies from `setup.py` are automatically included. 
+Dependencies to install prior to installing your package:
 
-Encode dependencies using [requirement specifiers](https://pip.pypa.io/en/stable/reference/pip_install/#requirement-specifiers).
+```toml
+test-deps = [
+    "pip-package",
+    "//in-repo-package"
+]
+```
+
+##### pip package
```

Comment:
the distinction between these two sections isn't clear. test-deps is marked optional. then we have this section called pip package which isn't the name of a section or marked optional

Reply:
The  test-deps header had `h4` styling, and the children had `h5`. I re-wrote it, though, to remove the subheaders, and I think it's clearer.